Protect local subnets from being routed toward Tailscale subnets if they collide (#201)
hassio-addons/addon-tailscale: tailscale/DOCS.md

execute steps 2 and 3 as described on [Site-to-site
networking][tailscale_info_site_to_site].

> In case your local subnets collide with subnet routes within your tailnet, your
> local network access has priority and these addresses won't be routed toward
> your tailnet. This will prevent your Home Assistant instance to lose network
> conection. This also means that using the same subnet on multiple nodes for load
> balancing and failover is not possible with the current add-on behavior.

Reviewer: ```suggestion
In case your local subnets collide with subnet routes within your tailnet, your
local network access has priority, and these addresses won't be routed toward
your tailnet. This will prevent your Home Assistant instance from losing network
connection. This also means that using the same subnet on multiple nodes for load
balancing and failover is impossible with the current add-on behavior.
```
